source/core/Console.hpp

```diff
@@ -181,6 +189,10 @@ namespace GTA
 		static const int VersionWidth = 50;
 		static const int InputHeight = 20;
 		static const float DefaultScale = 0.35f;
+		
+		static System::String ^UpdateCheckUserAgent = "scripthookvdotnet"; //Oh my dear github-api, why you do this...
```

Comment:
A HTTP user agent usually contains a version number, in the format `product/version`. We could do that too, e.g. `scripthookvdotnet/3.0`, via the following:
```c#
UpdateCheckUserAgent = "scripthookvdotnet/" + Console::typeid->Assembly->GetName()->Version->ToString(2);
```